bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -136,6 +147,11 @@ message GetBookieInfoRequest {
     optional int64 requested = 1;
 }
 
+message SyncRequest {
+   required int64 ledgerId = 1;
+   required bytes masterKey = 2;
```

Comment:
why do we need `masterKey` here?

Reply:
you are right, it does not make sense.

Reply:
Rollback. Thisi was needed, in order to access ledger impl on bookie side. See third part